### DISCUSSION TITLE
Working on Select Card, best option to compensate for select element height

### DISCUSSION THREAD

**Discussion by PaoloTK** [piitaya/lovelace-mushroom#883] [Ideas]
I'm working on a Mushroom Select Card for select and input_select entities. Unlike other mushroom control surfaces (e.g. sliders) selects already have a standard height value, 56px. Mushroom cards are designed to work with controls that are 42px tall. There are a few options to get around this problem, which one do you think is best? Select card is on the right in the screenshots.

1. [Increase the card height.](https://i.imgur.com/TLg5YrB.png) With this option, the card no longer aligns horizontally with other mushroom cards. In a horizontal-stack card this is not a problem because one can use "Fill container" on the other mushroom cards, but if the cards are added on their own they will be misaligned.
2. [Reduce the margins of the card.](https://i.imgur.com/NDcKcPo.png) With this option, the card is the same height as all the other mushroom cards, but it has thinner margins.
3. [Reduce the height of the select element.](https://i.imgur.com/O0rQzn6.png) With this option, the card is the same height as all the other mushroom cards, but the select element is shorter and thus different from all the others in HA.

**Comment by piitaya** [comment]
Hello 👋
I don't think we should add a dropdown to the UI. Button icon are better for that. Dropdown should be in the more info dialog.

**Reply by PaoloTK** [reply]
> Hello 👋 I don't think we should add a dropdown to the UI. Button icon are better for that. Dropdown should be in the more info dialog.

HI and thank you for replying. Buttons are a great if one only has a few options to choose between. One of my use cases is to control a light effect, and with WLED having like 50 effects I don't think there's a better option than a dropdown menu in the dashboard.

**Reply by piitaya** [reply]
Do you use the 50 effects ? Allow user select 3-4 favorite effect and display them with a custom icon can be great don't you think? And then you can go to the more info dialog if you want precise effect.

**Reply by PaoloTK** [reply]
> Do you use the 50 effects ? Allow user select 3-4 favorite effect and display them with a custom icon can be great don't you think? And then you can go to the more info dialog if you want precise effect.

Yes, I do use a significant amount of effects, not all but more than 3-4.
Beside, at the moment the "more info" dialog is not user-friendly at all, and it also doesn't match Mushroom design. I've seen the demo at State of the Open Home for how it's going to look eventually and it looks amazing, but for the time being the less an inexperienced user has to use the "more info" dialog the better IMHO.
